Assets/Scripts/InGame/Game.cs

```diff
@@ -130,6 +130,8 @@ private void StartPoker()
         if(!PhotonNetwork.IsMasterClient)
             return;
         
+        // create game account /v1/game/create-account here
```

Comment:
@Created-for-a-purpose have you seen any integration point where we can create player account ?

Reply:
Not sure. Was thinking of this:
https://github.com/Hadid-Ali/HoldEmup-Poker/blob/b6e49e9c5e204b51d4d4ca53571f2c1ec0c1d702/Assets/Scripts/InGame/Player/NetworkPlayer.cs#L161

The nickname and player account id need to have some sort of mapping, so that player does not have to create account again? Or is it fine that new player account is created on every login?

Reply:
Yes, It should be a mapping. But for now leave it as it is (let's create new player account everytime). As eventually we are going to build an email onboarding flow. (Similar to how magic link works).